pkg/httputil/tracing.go

```diff
@@ -134,19 +138,25 @@ func TracingMiddleware(requestIDHeaderName string, fields logging.Fields, traceR
 			next.ServeHTTP(responseWriter, r) // handle the request
 
 			traceFields := logging.Fields{
-				"took":             time.Since(startTime),
-				"status_code":      responseWriter.StatusCode,
-				"sent_bytes":       responseWriter.ResponseSize,
-				"request_body":     presentBody(requestBodyTracer.bodyRecorder.Buffer),
-				"response_body":    presentBody(responseWriter.BodyRecorder.Buffer),
-				"response_headers": responseWriter.Header(),
+				"start_time":  startTime,
+				"end_time":    time.Now(),
```

Comment:
Why added that new field start_time? (and end_time)

Reply:
damm forgot to commit this change